### PR TITLE
Make zoom controls bigger and change the icons

### PR DESCRIPTION
**Pull request by na9da**
### What this PR does

Made the zoom controls bigger and changed the icons to suit the larger size. Also changed the zoom reset icon to home icon. This is to make the reset behavior more discoverable.

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ ] I've updated relevant documentation in `doc/`.
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/ReactViews/Map/Navigation/Items/ZoomControl.tsx [hunk 1]
```diff
@@ -222,15 +222,18 @@ const StyledZoomControl = styled(Box).attrs(props => ({
...
-  styledWidth: "24px",
-  styledHeight: "60px"
+  styledWidth: "32px"
 }))`
   border-radius: 100px;
   svg {
-    height: 10px;
-    width: 10px;
+    height: 20px;
+    width: 20px;
```
Comment (na9da) [inline lib/ReactViews/Map/Navigation/Items/ZoomControl.tsx:231]: Same dimensions as other map nav icons.

**Review by nf-s — APPROVED** [review]
Looks good!

Can you please update `CHANGES` before merging

**Review by nf-s — CHANGES_REQUESTED** [review]
Actually, I think the colors might be slightly different. All other icons are `#4d5766`

image

Is that intentional?

**Comment by na9da** [comment]
Fixed the icon colors.

**Review by nf-s — APPROVED** [review]

**Pull request merged**